### PR TITLE
setup backend.

### PR DESCRIPTION
**Pull request by bmarika**

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[run]
+branch=True
+source=minion
```
Comment (ngurenyaga) [inline .coveragerc:3]: Do we really need the `minion` app? Is that the real coverage source for this project?

Comment (bmarika) [inline .coveragerc:3]: This will change once real apps are created.

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+machine:
+  environment:
+    DB_NAME: circle_test
+    DB_USER: ubuntu
+    DJANGO_SETTINGS_MODULE: config.settings.test
+    DEBUG: 'True'
+  python:
+    version: 2.7.6
```
Comment (ngurenyaga) [inline circle.yml]: Why do we need to pin to Python 2.7.6?

File: config/settings/base.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+https://docs.djangoproject.com/en/1.6/ref/settings/
+"""
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.6/howto/deployment/checklist/
```
Comment (ngurenyaga) [inline config/settings/base.py]: Many of the "helpful comments" in the settings file can be removed;

Also - was the `startproject` run with Django 1.7 or 1.8?

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='mfl',
+    version='2.0.0',
+    description='Master falicity list',
+    long_description=(''),
```
Comment (ngurenyaga) [inline setup.py]: Write a `README` then inline it here; that pattern is used a lot in our utilities.

Also - fill in the other fields ( other than `url` and `install_requires` ).

File: tox.ini [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+[testenv]
+deps =
+    -rrequirements.txt
+commands =
+    flake8 --exclude=migrations {toxinidir}
+    coverage erase
+    coverage run manage.py test
+    coverage report --fail-under=100
+
+[nosetests]
```
Comment (ngurenyaga) [inline tox.ini]: Do you plan to use `nose`?

**Comment by bmarika** [comment]
Let me make the changes.

File: config/settings/base.py (hunk 3 above)
Comment (bmarika) [inline config/settings/base.py]: Should have been 1.8 corrected that

**Pull request merged**